t/page/needed.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../lib/page/needed'
+require 'minitest/autorun'
+require 'octokit'
+require 'pry'
+require 'vcr'
+
+describe 'Needed' do
+  VCR.configure do |config|
+    config.cassette_library_dir = 't/fixtures/vcr_cassettes'
+    config.hook_into :webmock
+    config.ignore_request do |req|
+      req[:uri].include? 'https://cdn.rawgit.com/everypolitician/everypolitician-data'
```

Comment:
Why is this required? As far as I can see none of the requests are hitting anything that matches this.